examples/js/renderers/SVGRenderer.js

```diff
@@ -87,14 +87,45 @@ THREE.SVGRenderer = function () {
 
 	this.setPixelRatio = function () {};
 
-	this.setSize = function ( width, height ) {
+	this.getSize = function ( target ) {
```

Comment:
Please implement this similar to `CSS3DRenderer`:

https://github.com/mrdoob/three.js/blob/7df06a0e4054247f87a7b5aa555ca97cd6b15662/examples/jsm/renderers/CSS3DRenderer.js#L93-L100

Nice and simple.

Reply:
I copied the WebGLRenderer model instead of CSS3DRenderer. Any reason to use an anonymous object instead of a Vector2?

Reply:
I don't think it's important to use `Vector2` here. A plain object that holds `width` and `height` is totally fine.

Reply:
Updated